softdevice_controller/CHANGELOG.rst

```diff
@@ -7,7 +7,18 @@ Changelog
    :local:
    :depth: 2
 
-All the notable changes to this project are documented on this page.
+All the notable changes to this project are documented in this file.
+
+Main branch
+***********
+
+All the notable changes included in the main branch are documented in this section.
+
+Bug fixes
+=========
+
+* Fixed an issue where the controller was using a non-zero randomness for the very first advertising event even after calling :c:func:`sdc_hci_cmd_vs_set_adv_randomness` with a valid ``adv_handle`` parameter (DRGN-18261).
```

Comment:
```suggestion
* Fixed an issue where the controller was using non-zero randomness for the first advertising event.
  This happened even after calling :c:func:`sdc_hci_cmd_vs_set_adv_randomness` with a valid ``adv_handle`` parameter (DRGN-18261).
```
Split to two sentences to increase readability.
Randomness is not countable. Alternatively, it could "using a non-zero random delay" or something along those lines if I've understood the function properly.